build.sbt

```diff
@@ -18,10 +18,10 @@ val awsSdkSNS              = "com.amazonaws"          %  "aws-java-sdk-sns"
 val awsSdkCloudWatch       = "com.amazonaws"          %  "aws-java-sdk-cloudwatch"     % awsSdkVersion
 val stubbornArtifact       = "com.krux"               %% "stubborn"             % "1.4.1"
 val metricsGraphite        = "io.dropwizard.metrics"  %  "metrics-graphite"     % "4.1.12.1"
+val metricsJvm             = "io.dropwizard.metrics"  %  "metrics-jvm"          % "4.1.12.1"
```

Comment:
is this required?

Reply:
maybe we can remove this if it's not needed

Reply:
That's there so we can report JVM metrics. I can remove that if we don't want to send those to CloudWatch.

Reply:
don't think we are reporting jvm metrics. I would recommend stay with original implementation or have a very simplified one with only metrics that we are using.

Reply:
OK cool. You are correct that we are not reporting JVM metrics to CloudWatch currently.

Reply:
I have removed JVM metric reporting.